bread/layout/base.py

```diff
@@ -1,10 +1,32 @@
 import htmlgenerator as hg
+from django.conf import settings
+from django.http import HttpResponse
+from django.template.context import _builtin_context_processors
+from django.utils.module_loading import import_string
 
 from bread.utils import pretty_modelname, resolve_modellookup
 from bread.utils.urls import reverse_model
 
 from ..formatters import format_value
 
+CONTEXT_PROCESSORS = tuple(
+    import_string(path)
+    for path in _builtin_context_processors
+    + tuple(settings.TEMPLATES[0]["OPTIONS"]["context_processors"])
+)
```

Comment:
We read the desired context_processors from the default template settings, which is in 99% of the cases what is relevant. In theory we might not even need the ```TEMPLATE``` settings, but there might be external apps which require at least one template engine to be configured, so we leave that setting as it was.